### ISSUE TITLE
Missing GL_TEXTURE_CUBE_MAP_ARRAY in TextureTarget3d enum

### ISSUE THREAD

**Issue by the-Radek** [opentk/opentk#993]
### Description

Enum `OpenTK.Graphics.OpenGL4.TextureTarget3d` does not contain a definition for `TextureCubeMapArray`.

According to [wiki](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glTexStorage3D.xhtml) the `glTexStorage3D` (`OpenTK.Graphics.OpenGL4.GL.TexStorage3D`) function's one possible target is `GL_TEXTURE_CUBE_MAP_ARRAY`. However `GL.TexStorage3D` target is `TextureTarget3d` enum and it has no member similar to `GL_TEXTURE_CUBE_MAP_ARRAY`.

### Example

I'm trying to create CubeMap array texture this way:
```c#
using System;
using System.Drawing;
using System.Drawing.Imaging;
using OpenTK.Graphics.OpenGL4;
using PixelFormat = System.Drawing.Imaging.PixelFormat;
using GL_PixelFormat = OpenTK.Graphics.OpenGL4.PixelFormat;
...

public int CreateCubemapArray(string[] imagePaths, int count)
{
    // Generate an array texture
    int texture = GL.GenTexture();
    GL.BindTexture(TextureTarget.TextureCubeMapArray, texture);
    GL.TexStorage3D(
        TextureTarget3d.TextureCubeMapArray, // does not work
        9, 
        SizedInternalFormat.Rgba8, 
        512, 512, 
        count);

    // Load images
    for (int i = 0; i < count * 6; i++)
    {
        LoadImage(imagePaths[i], 512, 512, TextureTarget.TextureCubeMapPositiveX + (i % 6), i / 6);
    }

    // Setting behavior
    GL.TexParameter(TextureTarget.TextureCubeMapArray, TextureParameterName.TextureMinFilter, (int)TextureMinFilter.Linear);
    GL.TexParameter(TextureTarget.TextureCubeMapArray, TextureParameterName.TextureMagFilter, (int)TextureMagFilter.Nearest);

    GL.TexParameter(TextureTarget.TextureCubeMapArray, TextureParameterName.TextureWrapS, (int)TextureWrapMode.ClampToEdge);
    GL.TexParameter(TextureTarget.TextureCubeMapArray, TextureParameterName.TextureWrapT, (int)TextureWrapMode.ClampToEdge);
    GL.TexParameter(TextureTarget.TextureCubeMapArray, TextureParameterName.TextureWrapR, (int)TextureWrapMode.ClampToEdge);

    // Generate Mipmaps
    GL.GenerateMipmap(GenerateMipmapTarget.TextureCubeMapArray);

    return texture;
}

void LoadImage(string path, int width, int height, TextureTarget textureTarget, int depth)
{
    using Bitmap image = new Bitmap(path);
    BitmapData imageData = image.LockBits(
        new Rectangle(0, 0, image.Width, image.Height),
        ImageLockMode.ReadOnly,
        PixelFormat.Format32bppArgb);
    GL.TexSubImage3D(
        textureTarget,
        0, // level
        0, 0, 0, // offsets
        width, height,
        depth,
        GL_PixelFormat.Bgra,
        PixelType.UnsignedByte,
        imageData.Scan0);
    image.UnlockBits(imageData);
}
```

### Related information 

* OpenTK 3.1.0
* .NET Core 3.0

**Comment by NogginBops** [comment]
This issue is pretty old and you might have figured out the workaround already.

The workaround is to use the `All` enum to find the enum value that you want and cast it to the enum group which lacks the enum value you are after.
So in this case `(TextureTarget3d)All.TextureCubeMapArray` is what you want as a workaround for this.

**Comment by NogginBops** [comment]
Unclear in this is still an issue that needs work for opentk 5. Adding the 5.0 label in case it is.

**Comment by NogginBops** [comment]
This will be fixed with 4.7.2!
